### DISCUSSION TITLE
Any other type of filter than dropdown?

### DISCUSSION THREAD

**Discussion by elfeffe** [rappasoft/laravel-livewire-tables#245] [Q&A]
I don't see it in the roadmap, do you plan to release date or check filters?
it will be great also if the filters can have a default value, a default selection.

**Comment by bdelamatre** [comment]
Will you open individual feature requests for the new filter types?

Defaults for filters are supported, but I don't see it documented currently. All filters are bound to the $filters property which is a multi-dimensional array. So doing something like this:

`
$filters = [
  'myfiltername' => 'mydefaultvalue',
];
`

Should do the trick. It would probably make sense to add a default value option to the filter definition itself though.

**Reply by elfeffe** [reply]
Nice!
Yes, documentation and live examples are needed. I know that I'm lazy, but most of the developers are, and as I see that you want to update your package and I assume you want people to use it, I recommend what I see is needed.
No problem, I will open requests for them.
